README.md

```diff
@@ -37,7 +37,6 @@
 | [redis](./images/redis) | `cgr.dev/chainguard/redis` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/redis.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/redis:latest) |
 | [ruby](./images/ruby) | `cgr.dev/chainguard/ruby` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/ruby.build.status.3.0.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/ruby:3.0)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/ruby.build.status.3.1.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/ruby:3.1)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/ruby.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/ruby:latest) |
 | [rust](./images/rust) | `cgr.dev/chainguard/rust` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/rust.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/rust:latest) |
-| [sdk](./images/sdk) | `cgr.dev/chainguard/sdk` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/sdk.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/sdk:latest) |
```

Comment:
+1 to moving the image, but this raises the question of what "stable" means, if we can just remove an image labelled as stable.

To be clear, I'm not advocating _not_ doing this move, just saying we might need to have a clearer communication around these labels.